docs/RELEASING.md

```diff
@@ -1,221 +1,30 @@
 # Releasing D2-ui packages
 
-## TL;DR
+The root-level `package.json` contains the `version` key which will be
+used to publish all packages.
 
-```
-$ yarn install
-$ yarn release
-$ lerna publish
-```
-
-## Step 0: Before starting
-
-Make sure that you have run the `install` process to make sure that
-everything works as intended. Also double check the example application
-that everything looks OK.
-
-## Step 1: Building
-
-There is a shortcut command to run the `bootstrap`, `build`,
-`build:umd`, and `test`, commands in sequence:
-
-```
-yarn release
-```
-
-## Step 3: Publish
-
-Familiarise yourself with [Lerna](http://lernajs.io/) before proceeding.
-
-You will need it installed globally.
-
-Before running the following lerna commands, make sure you have the latest git tags locally
-```
-git fetch --tags
-```
+The release process has been simplified as to be executed by Travis
```

Comment:
The release process is executed by Travis using [deploy build]...

(can drop the history part)